frontend/device/pocketbook/powerd.lua

```diff
@@ -1,18 +1,37 @@
 local BasePowerD = require("device/generic/powerd")
 local ffi = require("ffi")
--- local inkview = ffi.load("inkview")
+local inkview = ffi.load("inkview")
 
 ffi.cdef[[
-int IsCharging();
```

Comment:
I don't know Pocketbook but could you clarify this removal?

Reply:
I removed It, because ``inkview.IsCharging()`` is not used anywhere. See ``PocketBookPowerD:isChargingHW()`` - it could be used there (and it may be more consistent to use inkview-library-calls for as much as possible), but currently is not.